wfe/web-front-end_test.go

```diff
@@ -870,21 +875,15 @@ func TestNewRegistration(t *testing.T) {
 	test.AssertEquals(t, responseWriter.Code, 409)
 }
 
-// Valid revocation request for existing, non-revoked cert
-func TestRevokeCertificate(t *testing.T) {
-	keyPemBytes, err := ioutil.ReadFile("test/238.key")
-	test.AssertNotError(t, err, "Failed to load key")
-	key, err := jose.LoadPrivateKey(keyPemBytes)
-	test.AssertNotError(t, err, "Failed to load key")
-	rsaKey, ok := key.(*rsa.PrivateKey)
-	test.Assert(t, ok, "Couldn't load RSA key")
-	signer, err := jose.NewSigner("RS256", rsaKey)
-	test.AssertNotError(t, err, "Failed to make signer")
-
+func makeRevokeRequestJSON(t *testing.T) ([]byte, error) {
 	certPemBytes, err := ioutil.ReadFile("test/238.crt")
-	test.AssertNotError(t, err, "Failed to load cert")
+	if err != nil {
+		return nil, err
```

Comment:
That's what I did at first: AssertNotError's everywhere. But it seemed cleaner for this function to not depend on a testing.T, and just return an error, which the caller can AssertNotError on. The advantage here is there's just the one consolidated error message "Failed to make revokeRequestJSON".

That said, I forgot to take out the testing.T arg. I can take it out, or I can switch to calling AssertNotError a bunch of times in makeRevokeRequestJSON. Which do you like better?